postgres.go

```diff
@@ -25,6 +27,93 @@ type DBConf struct {
 	clientKey  string
 }
 
+// Basebackup function:
+// - gets an identical copy of the pg database (pg_data)
+// - verifies the backup
+// - tars the copy
+// - compresses the encrypted file
+// - gets the key and encrypts the tar file
+// - puts the encrypted and compressed file in S3
+func (db DBConf) basebackup(sb s3Backend, keyPath string) error {
+	today := time.Now().Format("20060102150405")
+	destDir := "db-backup"
+	dbURI := buildConnInfo(db)
+	cmd := exec.Command("pg_basebackup", dbURI, "-F", "p", "-D", destDir)
+
+	var errMsg bytes.Buffer
+	cmd.Stderr = &errMsg
+
+	err := cmd.Run()
+	if err != nil {
+		log.Errorf(errMsg.String())
+		return err
+	}
+
+	cmd = exec.Command("pg_verifybackup", destDir)
+
+	cmd.Stderr = &errMsg
+
+	err = cmd.Run()
+	if err != nil {
+		log.Errorf(errMsg.String())
+		return err
+	}
+
+	cmd = exec.Command("tar", "-cvf", destDir+".tar", destDir)
+
+	cmd.Stderr = &errMsg
+
+	err = cmd.Run()
+	if err != nil {
+		log.Errorf(errMsg.String())
+		return err
+	}
+
+	fileName := today + "-" + db.database + ".tar"
```

Comment:
```suggestion
	fileName := today + "-" + db.database + ".enc"
```
It's better to use another suffix than `tar` since it is both compressed and encrypted.